### ISSUE TITLE
Clarification on potential values for lldp/interfaces/interface/neighbors/neighbor/state/management-address-type

### ISSUE THREAD

**Issue by rski** [openconfig/public#210]
Looking at the leaf, the description says the following:
```
    leaf management-address-type {
      type string;
      description
        "The enumerated value for the network address type
        identified in this TLV. This enumeration is defined in the
        'Assigned Numbers' RFC [RFC3232] and the
        ianaAddressFamilyNumbers object.";
    }
```

Are the expected values from https://www.iana.org/assignments/ianaaddressfamilynumbers-mib/ianaaddressfamilynumbers-mib, e.g. `ipV4` for IPv4? Or does "enumerated value" imply setting the leaf to `1` for IPv4?

**Comment by bstreiff** [comment]
I also have this question. I interpreted that description to mean that the "enumerated value" is `1` for IPv4 and `2` for IPv6, but admittedly that would fit better with an integer type instead of a `string`. Perhaps it makes more sense for this to be a [`iana-routing-types:address-family`](https://github.com/YangModels/yang/blob/master/standard/ietf/RFC/iana-routing-types%402017-12-04.yang#L45) in future revisions?

**Comment by aashaikh** [comment]
We did discuss this and agree that the description is confusing and needs to be updated.  Using the enitirety of the IANA addr family list also seems to be overkill given that platforms generally support IPv4 or IPv6 (and MAC address if no mgmt addr is specified).  Using the suggested iana module also creates external dependencies which we've tried to avoid.

So consideration is to change this to an enumeration type with IPV4, IPV6, MAC, and we could specify corresponding values that are consistent with IANA numbering (1,2,6, respectively).
